### DISCUSSION TITLE
[功能建议]: 关于搜索优先级排序的想法

### DISCUSSION THREAD

**Discussion by xwndo** [jxxghp/MoviePilot#3488] [讨论]
### 当前程序版本

v1.8.8

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

搜索优先级能否在增加一个维度，使用2个维度来排序，就是在现在的基础上在增加一个维度，比如把字幕和国语和60fps单独拿出来，因为，比如匹配”REMUX+4K+HDR或者杜比“这个资源肯定不止一个，然后在这1组里，把这1组在根据第二个维度“字幕和国语和60fps”这些按照权重依次排序，比如我看中特效字幕就特效字幕优先，我看中国语就国语优先。

这样对每一组在进行一次排序我感觉应该是不难实现吧，还是基于现在这种框架下。

因为如果不多维度，就需要做成倍的匹配规则，下面是我现在做好的，比如我现在又想每一种资源在60fps优先，就又要每个在添加一个，那实在是太多了。

![截图_20240515143156](https://github.com/jxxghp/MoviePilot/assets/44762565/15360a4c-de10-4a13-974c-992a94c3b00f)

**Comment by Kev1nT** [comment]
希望还能添加剧集和电影的纬度，电影需要4K，剧集仅需要1080    目前的规则好像不行
